src/uwtools/logger.py

```diff
@@ -73,8 +73,8 @@ def __init__(self, name: str = None,
         # Add console handler for logger
         _handler = Logger.add_stream_handler(
             level=self.level,
-            _format=self.format,
-            colored_log=self.colored_log,
+            _format='%(message)s',
+            colored_log= False,
```

Comment:
I doubt you want to make this change for all Logging functionality, after taking a second look at this. I think having the original defaults is nice. You should be able to overwrite the default values for format and colored_log when you create the Logger object.

Reply:
OK, I can take care of #128 while I'm at it with that change